source/unified-test-format/unified-test-format.rst

```diff
@@ -382,6 +404,15 @@ The structure of this object is as follows:
   treat the comparison as not equal and skip the test. This includes errors that
   occur when fetching a single parameter using ``getParameter``.
 
+- ``loadBalanced``: Optional boolean. Specifies whether or not the tests can
+  be executed against a load-balanced cluster. If true, the relevant tests MUST
+  only be run against a load balancer. If false or omitted, the tests MUST only
+  be run against non-load balanced clusters.
+
+- ``auth``: Optional boolean. Specifies whether or not authentication must be
+  enabled for the tests to run successfully. If this field is omitted, there is
+  no authentication requirement for the tests.
```

Comment:
Reading it, I'd take this as:
* `auth` missing: always run test
* `auth: true`: only run test when auth is enabled
* `auth: false`: only run test when auth is disabled

Is that assumption correct, or does `auth: false` mean that the test does not require auth but may very well run against a cluster with authentication enabled?

Reply:
I'd be in favor of having `auth: false` require auth to be disabled. If that was the original intention of this text, then I think we can just improve it to clarify (as was done for `loadBalanced`).

Reply:
Your understanding is correct and that was the original intention of the text. I updated the wording to clarify.

Reply:
Thank you for clarifying!